drivers/platform/xilinx/spi.c

```diff
@@ -189,7 +189,6 @@ int32_t spi_init(struct spi_desc **desc,
  */
 int32_t spi_remove(struct spi_desc *desc)
 {
-	int32_t			ret;
```

Comment:
Are you sure?
#ifdef XSPI_H
   ret = XSpi_Stop((XSpi *)(xdesc->instance));
   if(ret != SUCCESS)
      goto error;
#endif

Reply:
Well seen. I edited the commit.